.github/workflows/cd.yml

```diff
@@ -28,44 +38,56 @@ jobs:
       fail-fast: false
       matrix:
         platform: [iOS, macOS]
-        uploadto: [app-store, ftp]
     runs-on: macos-13
 
     steps:
+
     - name: Checkout code
       uses: actions/checkout@v4
 
-    - name: Set iOS extra xcode params
-      if: matrix.platform == 'iOS'
-      run: echo "EXTRA_XCODEBUILD=-sdk iphoneos ${{ env.APPLE_AUTH_PARAMS }}" >> $GITHUB_ENV
+    - name: Set VERSION from code
+      run: |
+        pip install pyyaml
+        python .github/workflows/version_from_code.py
```

Comment:
There's no reason to use a separate file for a simple task. Just use a python step

Reply:
please see it updated.